steps/cache-pull/2.2.0/step.yml

```diff
@@ -0,0 +1,103 @@
+title: Bitrise.io Cache:Pull
+summary: Download the build cache from bitrise.io.
+description: |-
+  Downloads the build cache from bitrise.io, and moves the files back to
+  their original place. The aim is to speed up your builds as not having to download every single dependency every time can be a serious timesaver.
+
+  ### Configuring the Step
+
+  To be able to pull from a previous build's cache, you need two things.
+
+  1. The **Cache:Push** Step at the end of your Workflow.
+  2. At least one successful build on the same branch in the last seven days.
+
+  If there was no successful build on the branch in question in the last seven days, the cache automatically gets deleted.
+
+  ### Troubleshooting
+
+  If the Step fails, check the time of the latest build. If there was a successful build in the last seven days, check if the Workflow included the **Cache:Push** Step.
+
+  If the **Cache:Push** Step was present, check the inputs of the Step. It's possible, for example, that the provided cache path is incorrect.
+
+  ### Useful links
+
+  - [Caching](https://devcenter.bitrise.io/builds/caching/about-caching-index/)
+
+  ### Related Steps
+
+  - [Cache:Push](https://www.bitrise.io/integrations/steps/cache-push)
+  - [Run CocoaPods install](https://www.bitrise.io/integrations/steps/cocoapods-install)
+  - [Run npm command](https://www.bitrise.io/integrations/steps/npm)
+  - [yarn](https://www.bitrise.io/integrations/steps/yarn)
+website: https://github.com/bitrise-steplib/steps-cache-pull
+source_code_url: https://github.com/bitrise-steplib/steps-cache-pull.git
+support_url: https://github.com/bitrise-steplib/steps-cache-pull/issues
+published_at: 2020-10-27T10:39:35.056615129Z
+source:
+  git: https://github.com/bitrise-steplib/steps-cache-pull.git
+  commit: 7b48d11cc95b594f14f75df29fc4aa7c7b5a7d95
+host_os_tags:
+- osx-10.10
+- ubuntu-14.04
+type_tags:
+- utility
+toolkit:
+  go:
+    package_name: github.com/bitrise-steplib/steps-cache-pull
+deps:
+  apt_get:
+  - name: tar
+is_requires_admin_user: false
+is_always_run: false
+is_skippable: true
+run_if: .IsCI
+inputs:
+- opts:
+    description: Working directory path - should be an absolute path.
+    summary: Working directory path
+    title: Working directory path
+  workdir: $BITRISE_SOURCE_DIR
+- cache_api_url: $BITRISE_CACHE_API_URL
+  opts:
+    description: Cache API URL
+    is_dont_change_value: true
+    summary: Cache API URL
+    title: Cache API URL
+- is_debug_mode: "false"
+  opts:
+    description: If debug mode is enabled, the step will print verbose logs
+    is_required: true
+    summary: If debug mode is enabled, the step will print verbose logs
+    title: Debug mode?
+    value_options:
+    - "true"
+    - "false"
```

Comment:
can we call it verbose logging? 🤔

Reply:
sure, it would be great to unify this kind of inputs, do we have something written about it?

Reply:
that's a good idea as well to unify this but I was thinking more here about using a name which reflects the functionality the best. Eg.  I don't know what happens exactly when I try to set `debug_mode`  but I know exactly what happens when I set `verbose_logging`.

Reply:
Let's change the title for this and then it's ok for now.